docs/unsupported_syntax.md

```diff
@@ -78,15 +78,22 @@ Examples of supported namespace syntax can be seen in the test fixture [tests/fi
 
 ### `module` namespace declarations
 
-`ts-blank-space` only erases TypeScript's `module` namespace declarations if they are marked with `declare` (see [`declare` hazard](#the-declare--hazard)).
+`ts-blank-space` only erases TypeScript's `module` statements if they are [ambient module augmentations](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation).
```

Comment:
This is incorrect. The TS doc even say these are easy to confuse 😂

It should say Ambient Module Declarations.

https://www.typescriptlang.org/docs/handbook/modules/reference.html#ambient-modules

Reply:
I think we are both correct. When the input is ESM then we are erasing ambient module augmentations. When the input is a script we are erasing ambient module declarations.

Reply:
Aha I did not know that.  Both terms are legit.  Let's lead by calling them ***Ambient Modules***.  The link describes both form.

```suggestion
`ts-blank-space` only erases TypeScript's `module` statements if they represent [Ambient Modules](https://www.typescriptlang.org/docs/handbook/modules/reference.html#ambient-modules) and applies to both _Ambient Module Augmentations_ as well as _Ambient Module Declarations_.
```

The link describes both forms:  Augmentations and Declarations.